src/components/reusable/modal/modal.scss

```diff
@@ -11,7 +11,7 @@ dialog {
   background: var(--kd-color-background-ui-default);
   border-radius: 8px;
   border: 1px solid var(--kd-color-border-light);
-  box-shadow: 0px 0px 40px 0px rgba(0, 0, 0, 0.12);
+  box-shadow: 0px 2px 12px 0px rgba(61, 60, 60, 0.25);
```

Comment:
You can use **elevation** here for box-shadow:
https://kyndryl-design-system.github.io/shidoka-foundation/?path=/docs/foundation-elevation--docs